tools/integration_tests/operations/operations_test.go

```diff
@@ -98,7 +99,7 @@ func createMountConfigsAndEquivalentFlags() (flags [][]string) {
 
 	// Set up config file with create-empty-file: true.
 	mountConfig1 := config.MountConfig{
-		WriteConfig: config.WriteConfig{
+		WriteConfig: cfg.WriteConfig{
```

Comment:
I'd rather we don't change the integration tests as much as possible. I'd like to run integration tests without changes after the migration is complete and ensure that the changes are backward compatible.

Reply:
Makes sense but this is the easiest way to migrate the old config to new config. I'll raise a follow up PR for all the integration tests so that they don't rely on config object to create yaml file.